src/scripts/reset.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# Reset the local Drupal site with a dump from the Backup/Restore files.
+#
+if [ -z "$1" ]; then
+  php drush bamls --files=private_files
+  exit
+fi
+
+drush bamr default_db private_files $1
```

Comment:
don't we need to sync the config here as in workbc-main ?

Reply:
probably.  To be honest, I never used this script. drush bamr always seems to log me out and caused me more trouble than just doing it manually

Reply:
Cool, never mind then. I will update the script as needed.

Yes, `drush bamr` logs you out because it nukes the sessions table.